src/test/java/org/jongo/marshall/jackson/JacksonMapperTest.java

```diff
@@ -102,7 +102,7 @@ public void canAddModule() throws Exception {
         ObjectId oid = new ObjectId("504482e5e4b0d1b2c47fff66");
         Friend friend = new Friend(oid, "Robert");
         Mapper mapper = jacksonMapper()
-                .registerModule(new Module() {
+                .registerModule(new com.fasterxml.jackson.databind.Module() {
```

Comment:
Fyi, you no longer need this now that you're using JDK8 to build, but it's harmless.